src/librustc/middle/traits/select.rs

```diff
@@ -1471,6 +1471,11 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
                     return Ok(ParameterBuiltin);
                 }
 
+                // Whether upvars are sized is checked elsewhere
```

Comment:
They are checked in numerous places... @nikomatsakis, do you have better ideas how to word this?